pr_watch/tasks.py

```diff
@@ -49,7 +49,7 @@ def watch_pr():
     organization on the watched repository
     """
     try:
-        for watched_fork in WatchedFork.objects.all():
+        for watched_fork in WatchedFork.objects.filter(enabled=True):
```

Comment:
Thanks for spotting this @clemente !

Just needs a test added to https://github.com/open-craft/opencraft/blob/master/pr_watch/tests/test_tasks.py